physical/postgresql_test.go

```diff
@@ -0,0 +1,42 @@
+package physical
+
+import (
+	"os"
+	"testing"
+
+	_ "github.com/lib/pq"
+)
+
+func TestPostgreSQLBackend(t *testing.T) {
+	connURL := os.Getenv("PGURL")
```

Comment:
Is `PGURL` a conventional variable for apps connecting to PG? I only ask as it seems vault tends towards using underscore delimited names for its environment variables, so it may help to stay consistent with that.